From 75c63fd063c8eaa035e2cbedf48bcbff3d9eea87 Mon Sep 17 00:00:00 2001
From: Andrew Jiang <Andrew.Jiang@amd.com>
Date: Fri, 6 Oct 2017 18:02:23 -0400
Subject: [PATCH 4139/5855] drm/amd/display: Don't set cursor address is 0
 logging as errors

This actually happens quite a bit, and having it as an error causes
false positive messages when running tests.

Signed-off-by: Andrew Jiang <Andrew.Jiang@amd.com>
Reviewed-by: Tony Cheng <Tony.Cheng@amd.com>
Acked-by: Harry Wentland <Harry.Wentland@amd.com>
---
 drivers/gpu/drm/amd/display/dc/core/dc_stream.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/amd/display/dc/core/dc_stream.c b/drivers/gpu/drm/amd/display/dc/core/dc_stream.c
index 97f040a..5cf69af 100644
--- a/drivers/gpu/drm/amd/display/dc/core/dc_stream.c
+++ b/drivers/gpu/drm/amd/display/dc/core/dc_stream.c
@@ -190,7 +190,7 @@ bool dc_stream_set_cursor_attributes(
 	}
 
 	if (attributes->address.quad_part == 0) {
-		dm_error("DC: Cursor address is 0!\n");
+		dm_output_to_console("DC: Cursor address is 0!\n");
 		return false;
 	}
 
-- 
2.7.4

